memory_statistics/memory_statistics.py

```diff
@@ -28,6 +28,10 @@ def make(sources, includes, flags, opt):
     (results, _) = proc.communicate()
     print(results)
 
+    args = ['make', '-f', os.path.join(__THIS_FILE_PATH__, 'makefile'), 'clean']
```

Comment:
What is the impact of these changes?

Reply:
It removes the FreeRTOSConfig.h that was used with every build. It wasn't actually used, so this removes it.